libusb/os/linux_usbfs.c

```diff
@@ -2770,8 +2770,16 @@ static int op_handle_events(struct libusb_context *ctx,
 	return r;
 }
 
+static const char* op_get_driver_name(void) {
+#if defined(HAVE_LIBUDEV)
+	return "usbfs with libudev";
+#else
+	return "usbfs with netlink";
+#endif
+}
+
 const struct usbi_os_backend usbi_backend = {
-	.name = "Linux usbfs",
+	.name = "Linux",
```

Comment:
is this relevant?

Reply:
My understanding is that `usbfs` is the name of the driver and `Linux` is the name of the backend. Since this PR adds somewhere to retrieve the driver name, it should be there instead of in the backend name.

Reply:
And yet, the `name` is claimed to be `A human-readable name for your backend, e.g. "Linux usbfs"` (as per documentation).
If backend is using a specific driver as implementation detail - it might as well be shown as part of human-readable name/description string.